store/types/store.go

```diff
@@ -1,16 +1,18 @@
 package types
 
 import (
+	"bytes"
 	"fmt"
 	"io"
 
 	dbm "github.com/cosmos/cosmos-db"
 	abci "github.com/tendermint/tendermint/abci/types"
 
-	"github.com/cosmos/cosmos-sdk/store/internal/kv"
 	"github.com/cosmos/cosmos-sdk/store/metrics"
+
 	pruningtypes "github.com/cosmos/cosmos-sdk/store/pruning/types"
 	snapshottypes "github.com/cosmos/cosmos-sdk/store/snapshots/types"
+	"github.com/cosmos/cosmos-sdk/types/kv"
```

Comment:
We shouldn't introduce back dependencies to the sdk in store.

Reply:
should it be replaced by the `Pair` in `store/internal/kv` ?

Reply:
this needs to stay like this but the locatin needs to be changed. I can handle this in a follow up pr.

Reply:
fixed